storage/src/Database.cpp

```diff
@@ -19,15 +19,13 @@ Database::~Database() {
     sqlite3_close(db);
 }
 
-Database::Database(boost::asio::io_context& ioc, const std::string& db_path,
-                   std::chrono::milliseconds cleanup_period)
-    : cleanup_period(cleanup_period), cleanup_timer_(ioc) {
+Database::Database(const std::filesystem::path& db_path) {
     open_and_prepare(db_path);
 
-    perform_cleanup();
+    clean_expired();
```

Comment:
Is this only called once now?

Reply:
Oops, forgot to including the corresponding timer in service_node.cpp in the commit.  Updated.